conn.go

```diff
@@ -11,8 +12,9 @@ import (
 )
 
 type Conn struct {
-	inet.Stream
+	stream inet.Stream
```

Comment:
I don't think you need to remove the embedding. You should be able to override the `Close()` method delegating the call to `c.Stream.Close()`.

Reply:
again, wrong PR; these are on the base pr.

Reply:
Taken to #76.